lib/orderbook/OrderBook.ts

```diff
@@ -541,6 +542,11 @@ class OrderBook extends EventEmitter {
     this.logger.debug(`removed all orders for peer ${peer.nodePubKey}`);
   }
 
+  private removePeerPair = (peerPubKey: string, pairId: string) => {
+    const tp = this.getTradingPair(pairId);
+    tp.removePeerOrders(peerPubKey);
```

Comment:
shouldn't this be similar to how `removePeerOrders` is being handled, and so to emit `peerOrder.invalidation` for every removed order?

Reply:
Yes you are right, good catch. I'll make a reusable method for this so it can be called in the future to make sure this event gets emitted.